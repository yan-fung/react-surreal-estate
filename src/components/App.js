import React from "react";
import NavBar from "./NavBar";
import "../styles/app.css";

const App = () => (
  <div className="App">
    <NavBar />
  </div>
);

export default App;
