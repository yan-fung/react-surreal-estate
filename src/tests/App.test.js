import React from "react";
import { render, screen } from "@testing-library/react";
import App from "../components/App";

it("renders correctly", () => {
  render(<App />);
  const linkElement = screen.getByText(/Surreal Estate/i);
  expect(linkElement).toBeInTheDocument();
});
