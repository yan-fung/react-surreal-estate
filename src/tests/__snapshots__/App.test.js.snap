// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders app correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="home_image"
        src="home.png"
      />
      <ul
        class="navbar-links"
      >
        <li
          class="narbar-links-item"
        >
          <a
            class="narbar-links__text"
            href="/"
          >
            View Properties
          </a>
        </li>
        <li
          class="narbar-links-item"
        >
          <a
            class="narbar-links__text"
            href="/add_property"
          >
            Add Properties
          </a>
        </li>
      </ul>
    </div>
    <div
      class="property_header"
    >
      <h1>
        Properties Page
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
